### ISSUE TITLE
--check-github fails with old GitPython versions

### ISSUE THREAD

**Issue by pescobar** [hpcugent/easybuild-framework#1757]
I tried `eb --github-user=pescobar --check-github` and I was getting this error:

```
Checking status of GitHub integration...

Making sure we're online... OK

* GitHub user... pescobar => OK
* GitHub token... fa1..5f5 (len: 40) => OK (validated)
* git command... OK ("git version 1.8.5.6; ")
* GitPython module... OK
* push access to pescobar/easybuild-easyconfigs repo @ GitHub... FAIL (unexpected exception: 'Remote' object has no attribute 'exists')
* creating gists... OK
* location to Git working dirs...  not found (suboptimal)
```

I was using python 2.6 and GitPython from centos6 epel repository `GitPython-0.3.2-0.6.RC1.el6.noarch`

After upgrading my GitPython installation doing `easy_install -U --prefix=$HOME/.local GitPython` the issue is fixed. It seems there is an problem with old GitPython versions. After the upgrade it's working:

```
All checks PASSed!

Status of GitHub integration:
* --from-pr: OK
* --new-pr: OK
* --review-pr: OK
* --update-pr: OK
* --upload-test-report: OK
```

**Comment by boegel** [comment]
for future reference: latest GitPython version is 2.0.2 (i.e. the one that @pescobar is using) at this time

**Comment by boegel** [comment]
fixed with #1847
